### PR TITLE
Added option for Arm NN delegate

### PR DESCRIPTION
**Pull request by wwwind**
Added option to configuration.proto for Arm NN delegate

### CODE REVIEW

File: tensorflow/lite/experimental/acceleration/configuration/configuration.proto [hunk 1]
```diff
@@ -570,6 +570,11 @@ message CPUSettings {
   optional int32 num_threads = 1 [default = -1];
 }
 
+// Arm NN delegate option
+message ArmNNSettings {
+  optional string delegate_options = 1;
```
Comment (fergushenderson) [inline tensorflow/lite/experimental/acceleration/configuration/configuration.proto:575]: Could you please document this field, or include a pointer to where developers can find documentation for it?

**Comment by gbaned** [comment]
Hi @wwwind Can you please check @fergushenderson's comments and keep us posted ? Thank you!

File: tensorflow/lite/experimental/acceleration/configuration/configuration.proto (hunk 1 above)
Comment (wwwind) [reply tensorflow/lite/experimental/acceleration/configuration/configuration.proto:576]: Hi @fergushenderson Could you please review it again ? The comment has been added. Sorry for the delay.

**Comment by fergushenderson** [comment]
It looks like
https://github.com/tensorflow/tensorflow/pull/60352
is a more recent and more complete version of this pull request.

**Comment by gbaned** [comment]
Hi @wwwind Can you please check @fergushenderson's comments and resolve conflicts?. Thank you!

**Review by fergushenderson — COMMENTED** [review]
This file has moved out of "experimental/", to tensorflow/lite/acceleration/configuration/configuration.proto.
Can you please merge from master again?

**Pull request closed without merging**